projects/v3/src/app/services/achievement.service.ts

```diff
@@ -39,64 +45,126 @@ export class AchievementService {
   isPointsConfigured = false;
 
   constructor(
+    private apolloService: ApolloService,
     private request: RequestService,
     private utils: UtilsService,
-    private demo: DemoService,
-  ) { }
+    private demo: DemoService
+  ) {}
+
+  /**
+   * [graphQLGetAchievements description]
+   * @link https://intersective.github.io/core-graphql-api/query.doc.html#:~:text=achievements(filter%3A%20String%2C%20type%3A%20String%2C%20active%3A%20Boolean)%3A%20%5BAchievement%5D
+   * @param   {string}                     filter  keyword to search for
+   * @param   {string}                     type    achievement type
+   * @param   {boolean}                    active  status of is active?
+   *
+   * @return  {Observable<Achievement>[]}          achievement list in badges
+   */
+  graphQLGetAchievements(
+    filter?: string,
+    type?: string,
+    active?: boolean
+  ): Observable<Achievement[]> {
+    return this.apolloService
+      .graphQLFetch(
+        `query achievements($filter: String, $type: String, $active: Boolean) {
```

Comment:
@shawnm0705 now using `query achievements`

Reply:
When we use `query achievements` in the APP, we don't need to worry about the arguments of this endpoint. They are all built for LearnFlow. We can just query it without any arguments